### PR TITLE
DOCS-2110: Update frame system screenshots for R2D2 editor

### PR DESCRIPTION
**Pull request by sguequierre**
* edits frame system service and reference frame docs 
* replaces screenshots
* cleans up service page a bit

### CODE REVIEW

File: docs/services/frame-system/_index.md [hunk 1]
```diff
@@ -42,10 +43,11 @@ You can configure a reference frame within the frame system for each of your mac
 2. From the left-hand menu, select your component.
    If you haven't adjusted any parameters yet, the default reference frame will be shown for the component:
 
-   {{<imgproc src="/services/frame-system/frame_card.png" resize="300x" style="width: 500px" alt="Frame card for a camera with the default reference frame settings">}}
+   {{<imgproc src="/services/frame-system/frame_card.png" resize="300x" style="width: 250px" alt="Frame card for a camera with the default reference frame settings">}}
 
 3. To adjust the frame from its default configuration, change the parameters as needed for your machine before saving.
-   Select a **Parent** frame and fill in the coordinates for **Translation** (_mm_) and **Orientation** (_deg_, _rad_, or _q_), according to the position and orientation of your component in relation to the **Parent** frame.
+   Select a **Parent** frame and fill in the coordinates for **Translation** (m) and **Orientation** (deg, rad, or q), according to the position and orientation of your component in relation to the **Parent** frame.
```
Comment (sguequierre) [inline docs/services/frame-system/_index.md:49]: unitalicized units bc i looked it up and apparently you're not supposed to do that

File: docs/services/frame-system/_index.md [hunk 2]
```diff
@@ -151,9 +153,9 @@ Configure the reference frame as follows:
 | Parameter | Required? | Required |
 | --------- | ----------- | ----- |
 | `parent`  | **Required** | Default: `world`. The name of the reference frame you want to act as the parent of this frame. |
-| `translation` | **Required** | Default: `(0, 0, 0)`. The coordinates that the origin of this component's reference frame has within its parent reference frame. <br> Units: _mm_. |
+| `translation` | **Required** | Default: `(0, 0, 0)`. The coordinates that the origin of this component's reference frame has within its parent reference frame. <br> Units: m in Frame Editor, mm in JSON. |
```
Comment (sguequierre) [inline docs/services/frame-system/_index.md:156]: this is indeed the case that the units are different in the editor-- confirmed with michael parks. a bit odd but he said he would make a ticket to mark units on the editor so at least that will be more clear

File: docs/services/frame-system/_index.md [hunk 3]
```diff
@@ -288,127 +265,13 @@ The resulting tree of reference frames looks like:
 
 ## Access the frame system
 
-The [Robot API](https://github.com/viamrobotics/api/blob/main/proto/viam/robot/v1/robot.proto) supplies the following method to interact with the frame system:
+The [Machine Management API](/appendix/apis/robot/) supplies the following methods to interact with the frame system:
 
 <!-- prettier-ignore -->
 | Method Name | Description |
 | ----- | ----------- |
```
Comment (sguequierre) [inline docs/services/frame-system/_index.md:272]: removed this in favor of linking to the methods (which we now have both documented) bc I think that makes more sense/leaves us not having to duplicate, lmk and I can put them back

File: docs/services/frame-system/_index.md [hunk 4]
```diff
@@ -189,71 +191,46 @@ On this tab, you can simultaneously view and edit the position, orientation, and
...
 However, the camera doesn't yet display as oriented towards the base.
 
 Adjust the [orientation vector](/internals/orientation-vector/) to 0.5 degrees in `"ox"` in the camera's reference frame `Orientation` attribute, and the frame system readjusts to show the camera's orientation:
 
-![Camera oriented .5 degrees OX shown in the Frame System Editor](/services/frame-system/demo_camera_edited_2.png)
+![Camera oriented 0.5 degrees OX shown in the Frame System Editor](/services/frame-system/demo_camera_edited_2.png)
 
-Now that the frame system is accurately configured with the machine's spatial orientation, [motion service](/services/motion/) methods that take in reference frame information can be used.
-
-### Display options
```
Comment (sguequierre) [inline docs/services/frame-system/_index.md:219]: removed this part bc it should be self explanatory in the app

File: docs/services/frame-system/_index.md (hunk 2 above)
Comment (npentrel) [reply docs/services/frame-system/_index.md:156]: good catch

File: docs/services/frame-system/_index.md (hunk 3 above)
Comment (npentrel) [reply docs/services/frame-system/_index.md:272]: nice

File: docs/services/frame-system/_index.md [hunk 5]
```diff
@@ -430,6 +293,6 @@ For example:
 
 Usage:
 
-- You can pass a detected object's frame information to the `supplemental_transforms` parameter in your calls to Viam's motion service's [`GetPose`](/appendix/apis/services/motion/#getpose) method.
+- You can pass a detected object's frame information to the `supplemental_transforms` parameter in your calls to Viam's motion service's [GetPose](/appendix/apis/services/motion/#getpose) method.
 - Functions of some services and components also take in a `WorldState` parameter, which includes a `transforms` property.
-- [`TransformPose`](#access-the-frame-system) has the option to take in these additional transforms.
+- [TransformPose](/appendix/apis/robot/#transformpose) has the option to take in these additional transforms.
```
Comment (npentrel) [inline docs/services/frame-system/_index.md:298]:
```suggestion
- [`TransformPose`](/appendix/apis/robot/#transformpose) has the option to take in these additional transforms.
```

**Pull request merged**